src/index.js

```diff
@@ -251,27 +239,38 @@ var MaskedInput = React.createClass({
       onPaste: this._onPaste,
       [this._keyPressPropName()]: this._onKeyPress
     }
-  },
+  }
 
   focus() {
     this.input.focus()
-  },
+  }
 
   blur() {
     this.input.blur()
-  },
+  }
 
   render() {
-    var ref = r => this.input = r
+    var ref = r => { this.input = r }
```

Comment:
should get all the `var`'s to `const` and `let` as appropriate while you're at it